### ISSUE TITLE
Wish: pgup/pgdn should scroll per page

### ISSUE THREAD

**Issue by jimkenith** [frescobaldi/frescobaldi#1303]

**Comment by jimkenith** [comment]
The smooth scroll with the mouse wheel or pgup/pgdn seems cumbersome for large documents. Perhaps an immediate scroll would be faster to use. Also, if the pdf is set to fit the container, pgup/pgdn would navigate by page, greatly speeding up navigation. (SumatraPDF style PDF navigation, very fast and simple,)

**Comment by uliska** [comment]
You can use the toolbar's navigation buttons to go page-wise. But I agree that a left/right shortcut would be nice.

The issue I see is that usually the music view doesn't have keyboard focus, so we might use somethinc else, like Meta+arrow.

**Comment by fedelibre** [comment]
> Smooth scroll is cumbersome to use when navigating large documents. Perhaps an option to disable smooth scroll would be beneficial.

I remember that there was an option to enable/disable kinetic scrolling, but I can't find it anymore in the Preferences. Is this a new bug?

```
$ git grep kinetic frescobaldi_app/preferences/
frescobaldi_app/preferences/musicviewers.py:        kineticScrollingActive = s.value("kinetic_scrolling", True, bool)
frescobaldi_app/preferences/musicviewers.py:        self.enableKineticScrolling.setChecked(kineticScrollingActive)
frescobaldi_app/preferences/musicviewers.py:        s.setValue("kinetic_scrolling", self.enableKineticScrolling.isChecked())
```

**Comment by fedelibre** [comment]
Ops, forget my previous comment. I found the preference for disabling the kinetic scrolling. It's in Preferences>Music Preferences (third). @jimkenith did you miss it?

I will update the title of this issue

**Comment by jimkenith** [comment]
@fedelibre Yes, I did miss it! I apologize, I am new to Lilypond and did not realize the option was under Music Preferences. Thank you very much.

**Comment by wbsoft** [comment]
We could add an option to scroll per page.

**Comment by jimkenith** [comment]
Awesome, thanks.

**Comment by fedelibre** [comment]
> We could add an option to scroll per page.

There is already! You should check *Preferences>Music Preferences>Strict Paging*. (I suggest disabling Kinetic Scrolling to get faster and smoother page changes)

@jimkenith You can simply click on the Music View to get focus, then use PageUp/PageDown to change page.

Let's close this issue.
